NuGet.config

```diff
@@ -7,16 +7,21 @@
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet-corefxtestdata" value="https://dotnet.myget.org/F/dotnet-corefxtestdata/api/v3/index.json" />
+
+    <!-- Microsoft.VisualStudio.* packages (e.g. Microsoft.VisualStudio.LanguageServer.Protocol.Extensions): https://github.com/dotnet/roslyn/issues/43242 -->
     <add key="experimental-vs-packages" value="https://dotnet.myget.org/F/experimental-vs-packages/api/v3/index.json" />
-    <add key="symreader-native" value="https://dotnet.myget.org/F/symreader-native/api/v3/index.json" />
-    <add key="interactive-window" value="https://dotnet.myget.org/F/interactive-window/api/v3/index.json" />
-    <add key="roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
+  
+    <!-- Microsoft.CodeAnalysis.Test.Resource.Proprietary -->
     <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+
     <add key="roslyn-analyzers" value="https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json" />
-    <add key="roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
-    <add key="vs-devcore" value="https://myget.org/F/vs-devcore/api/v3/index.json" />
+
+    <!-- Microsoft.VisualStudio.* packages (e.g. Microsoft.VisualStudio.ImageCatalog): https://github.com/dotnet/roslyn/issues/43242 -->
     <add key="vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
+
+    <!-- Microsoft.VisualStudio.ProjectSystem.Managed -->
+    <add key="roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
Any idea why this is in the Roslyn feed versus...something else?

Reply:
Historical reasons I guess. It should be VSSDK or vs-impl feed.

Reply:
Filed https://github.com/dotnet/project-system/issues/6079